packages/vmcloak/PKGBUILD

```diff
@@ -2,45 +2,52 @@
 # See COPYING for license details.
 
 pkgname=vmcloak
-pkgver=912.13853ff
-pkgrel=2
-epoch=2
+pkgver=955.fae1f29
+pkgrel=1
+epoch=3
 pkgdesc='Automated Virtual Machine Generation and Cloaking for Cuckoo Sandbox.'
-groups=('blackarch-malware')
-url='https://github.com/jbremer/vmcloak'
 arch=('any')
+groups=('blackarch' 'blackarch-malware')
+url='https://github.com/Cryss76/vmcloak'
```

Comment:
Could you explain how you chose this fork?

Reply:
Because of:
* https://github.com/hatching/vmcloak/issues/218
* the maintainer still seems to be active in the last month: https://github.com/Cryss76/vmcloak/issues/6
* note sure if there are better maintained `vmcloak` project